packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -406,14 +472,13 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
       }
 
       .mdc-textfield__label {
-        color: $mdc-textfield-error-on-dark;
+        @include mdc-theme-prop(color, $mdc-textfield-error-on-dark);
       }
     }
   }
 }
 
 .mdc-textfield--disabled {
-  border-bottom: 1px dotted $mdc-textfield-disabled-border-on-light;
   pointer-events: none;
 
   &::after {
```

Comment:
Given that bottom-line is now separate, is this style doing anything anymore?

Reply:
It is not